docs/operate/get-started/other-hardware/create-module/dependencies.md

```diff
@@ -287,6 +291,276 @@ There is not currently an SDK method to directly access configuration attributes
 You must access the API key module environment variables to establish the app client connection.
 {{% /hiddencontent %}}
 
+## Use SDK methods on dependencies
```

Comment:
@npentrel I added all this here (still untested code for now). Didn't delete the motion changes yet, just for staging reference in discussion, since this isn't merging yet anyway.